website/docs/GettingStarted.md

```diff
@@ -9,22 +9,21 @@ Welcome to the WebdriverIO documentation. It will help you to get started fast.
 These are the docs for the latest version (__>=7.x__) of WebdriverIO. If you are still using an older version, please visit the [old documentation websites](/versions)!
 :::
 
-## Installation
+## Installation considerations
 
-Use npm or Yarn to install the WebdriverIO test runner in your Node.js project. See [system requirements](#system-requirements).
+There are two ways of using WebdriverIO - you can either add it to the application you plan to test, or you can create a brand new NodeJS project. 
```

Comment:
If we mention that there are two ways to install wdio we should also clearly describe them. I think we can just use the starter toolkit `npx wdio .` and assume someone is currently in an existing project

Reply:
Well - I just did the Webdriver in a folder for an experiment. 
Perhaps also if you are doing an appium project on an ios and android project, then there is no existing nodejs project?

Reply:
You right. I think my concern was just about saying that there are two ways without explicitly documenting both